src/jvm/test/resources/expected_logs/obstruction_freedom_synchronized.txt

```diff
@@ -10,19 +10,21 @@ The following interleaving leads to the error:
 |                                                         Thread 1                                                         |                                                         Thread 2                                                         |
 | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 |                                                                                                                          | operation()                                                                                                              |
-|                                                                                                                          |   MONITORENTER at ObstructionFreedomSynchronizedRepresentationTest.operation(ObstructionFreedomRepresentationTest.kt:70) |
+|                                                                                                                          |   MONITORENTER at ObstructionFreedomSynchronizedRepresentationTest.operation(ObstructionFreedomRepresentationTest.kt:61) |
 |                                                                                                                          |   switch                                                                                                                 |
 | operation()                                                                                                              |                                                                                                                          |
-|   MONITORENTER at ObstructionFreedomSynchronizedRepresentationTest.operation(ObstructionFreedomRepresentationTest.kt:70) |                                                                                                                          |
+|   MONITORENTER at ObstructionFreedomSynchronizedRepresentationTest.operation(ObstructionFreedomRepresentationTest.kt:61) |                                                                                                                          |
+|   switch (reason: lock is already acquired)                                                                              |                                                                                                                          |
```

Comment:
is there any switch? It seems the execution ends after this event.

Reply:
Fixed